molecule/default/molecule.yml

```diff
@@ -220,6 +220,15 @@ platforms:
     privileged: true
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:rw
+provisioner:
+  name: ansible
+  env:
+    # Molecule used to add ${MOLECULE_PROJECT_DIRECTORY}/. to this
+    # path for us pre-25.2.0, but now we have to do it ourselves.  See
+    # ansible/molecule#4380 and
+    # https://github.com/ansible/molecule/releases/tag/v25.2.0 for
+    # more details.
+    ANSIBLE_ROLES_PATH: ~/.cache/molecule/skeleton-ansible-role/${MOLECULE_SCENARIO_NAME}/roles:${MOLECULE_PROJECT_DIRECTORY}/..:~/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles
```

Comment:
Do any of our roles run afoul of `ANSIBLE_COLLECTIONS_PATH` as well?

Reply:
I don't think so.  The default value of `ANSIBLE_COLLECTIONS_PATH` should suffice, since we don't create any collections.  For example, if I don't define `ANSIBLE_ROLES_PATH` then `molecule` still finds the version of `ansible-role-upgrade` that is pulled via `ansible-galaxy` during the `dependency` stage, it just can't find the local checkout of `skeleton-ansible-role`.

Still, I should probably set `ANSIBLE_COLLECTIONS_PATH` as well as `ANSIBLE_FILTER_PLUGINS` and `ANSIBLE_LIBRARY` just as `molecule` used to.  I will look into doing that.